### DISCUSSION TITLE
have an issue with windows key and tabbing out whilst in game

### DISCUSSION THREAD

**Discussion by Opus-greybeard** [valinet/ExplorerPatcher#3697] [Q&A] [answered]
After the latest update I installed yesterday I have the following issue : If Im in a game say Apex Legends and want to exit the game window because i need to lower the volume on my music player or any other given task I always hit my left windows button to "tab" out, when I do this fileexplorer opens up my "HOME" folder in a separate window and it keeps opening a new window everytime I hit the windows button.
The wierd thing is I can hit the windows button all I want when im out of the game and no "HOME" windows will open, only when I have an overlaying window like a active game. This never happened before, so must be update related,
I tabb out often during my gaming sessions so having to close 10-15 windows after im done is no biggie, only annoying
Help is gladly accepted.
Opus

**Comment by pyrates999** [comment, marked as answer]
does it happen if you uninstall EP and reboot?

**Reply by Opus-greybeard** [reply]
that did the trick, should have done it right away, but was worried about custom OS setup, once I decided to redo it all the problem went away, thx
